packages/interface/src/features/applications/components/ApplicationItem.tsx

```diff
@@ -10,41 +10,40 @@ import { useMetadata } from "~/hooks/useMetadata";
 import { formatDate } from "~/utils/time";
 
 import type { Application } from "~/features/applications/types";
-import type { Attestation } from "~/utils/types";
+import type { IRecipient, IRecipientContract } from "~/utils/types";
 
-export interface IApplicationItemProps extends Attestation {
+export interface IApplicationItemProps {
+  index: string;
+  recipient: IRecipient | IRecipientContract;
   isApproved?: boolean;
   isLoading?: boolean;
 }
 
 export const ApplicationItem = ({
-  id,
+  index,
```

Comment:
why rename `id` to `index`?

Reply:
Because there is no id on contracts side. There is only index and it's unique for the applications